turbinia/turbiniactl.py

```diff
@@ -670,6 +675,11 @@ def main():
           'The evidence type {0:s} is Cloud only, and this instance of '
           'Turbinia is not a cloud instance.'.format(evidence_.type))
       sys.exit(1)
+    elif not config.SHARED_FILESYSTEM and evidence_.copyable:
+      if os.path.exists(evidence_.local_path):
```

Comment:
Maybe bail out with an error here if the path doesn't exist?  Otherwise the processing request will fail.

Reply:
Added an else statement that would error out if it doesn't